bazel/rules_go.patch

```diff
@@ -28,3 +28,15 @@ index 91748eda..c1aeb91e 100644
              outputs = [out, gopath, gocache],
              mnemonic = "GoToolchainBinaryBuild",
          )
+diff --git a/go/private/platforms.bzl b/go/private/platforms.bzl
```

Comment:
Do you think it's worth to put a comment explaining why do we need this (and that this is specific to maistra)? Similar to the comment on the top of this file

Reply:
Yes it is specific to maistra. We can put a comment as below, let me know if this fine

> Proxy build failed for ppc64le, as ppc64(Big endian) is getting picked for cpu value which was added https://github.com/bazelbuild/rules_go/pull/3336 which is causing failures on ppc64le. And changes for rules_go, is still under review from community. https://github.com/bazelbuild/rules_go/issues/3626
> So adding temporary changes for maistra/proxy 2.5 build failures, which can be removed after resolving

Reply:
LGTM